man/systemd.unit.xml

```diff
@@ -2131,7 +2131,7 @@ Note that this setting is <emphasis>not</emphasis> influenced by the <varname>Us
           <row>
             <entry><literal>%L</literal></entry>
             <entry>Log directory root</entry>
-            <entry>This is either <filename>/var/log</filename> (for the system manager) or the path <literal>$XDG_CONFIG_HOME</literal> resolves to with <filename index="false">/log</filename> appended (for user managers).</entry>
+            <entry>This is either <filename>/var/log</filename> (for the system manager) or the path <literal>$XDG_STATE_HOME</literal> resolves to with <filename index="false">/log</filename> appended (for user managers).</entry>
```

Comment:
If you touch this, maybe suffix `/var/log` and `/log` with slash as what we do nowadays?

Reply:
Also, maybe change `<literal>` to `<varname>`.

Reply:
so, while I would usually agree with suffixing the slash here, here we really shouldn't. That's because `%L` is actually replaced by the path *without* a trailing slash, and this matters, since people will probably use this in a syntax like `ExecStart=/usr/bin/somecommand %L/foobar`. If we suggest here that `%L` actually resolves to `/var/log/` they might end up writing `ExecStart=/usr/bin/somecommand %Lfoobar` which would generate an invalid path.

Reply:
will make the literal → varname change though